openreview/conference/builder.py

```diff
@@ -1701,6 +1701,7 @@ def send_notification(note):
                 submission_title=paper_note.content['title']
             )
             final_message = f'''{message.format(
+                fullname='{{fullname}}',
                 submission_title=paper_note.content['title'],
```

Comment:
if submission_title has double curly braces, does the format remove all of them?

Reply:
Could change the assertion here: https://github.com/openreview/openreview-py/pull/1268/files#diff-e3ae2f10352929825bba9040318c444e8bf4b97395b036eec532aacae837bf72R1845

and assert the whole message? then you can verify all the token where correctly replaced.

Reply:
This is what I suspected. I think we should use find and replace instead of format.

Reply:
But we will always get single curly braces, right?

Reply:
The UI should show all the tokens with double curly braces so they match with the backend supported tokens ({{firstname}} and {{fullname}}). 

Now I'm seeing single curly braces. 

<img width="978" alt="Screen Shot 2022-05-19 at 9 50 59 AM" src="https://user-images.githubusercontent.com/545506/169309722-f07b8747-716b-40f1-8733-db1477d10bc2.png">